README.md

```diff
@@ -105,6 +108,27 @@ require'nvim-treesitter.configs'.setup {
 EOF
 ```
 
+Again, you can also map captures from other query groups
+e.g. cycle through function names defined in `locals.scm`.
+
+
+```lua
+lua <<EOF
+require'nvim-treesitter.configs'.setup {
+  textobjects = {
+     move = {
+      enable = true,
+      goto_next_start = {
+        ["]F"] = { "@definition.function", "locals" },
+      },
+      goto_previous_start = {
+        ["[F"] = { "@definition.function", "locals" },
```

Comment:
@synek @tom-anders this config hijacks the queries in `locals.scm` to be used as textobject. It would also be possible to fold on textobjects. We already have classes, functions, etc. Next, step would be to allow `@function.outer` *and* `@class.outer` at the same time.